### PR TITLE
Add a check if there is nothing to be piped

### PR DESCRIPTION
**Pull request by p-v**

### CODE REVIEW

**Comment by viktorbenei** [comment]
Related issue: https://github.com/huytd/go-play/issues/15 (just for the link between the two ;)

File: main.go [hunk 1]
```diff
@@ -3,24 +3,35 @@ package main
...
 	"flag"
 	"fmt"
+	"github.com/huytd/go-play/engine"
+	"github.com/huytd/go-play/web"
 	"io/ioutil"
 	"log"
 	"os"
-
-	"github.com/huytd/go-play/engine"
-	"github.com/huytd/go-play/web"
```
Comment (pawelkowalak) [inline main.go:11]: Bad imports grouping.

Comment (p-v) [inline main.go:11]: It's not bad in the pr. It's the version from the main branch you are seeing.

Comment (pawelkowalak) [inline main.go:11]: In your commit `github.com/huytd/go-play` imports are together with standard library imports, but should be separated :) https://github.com/golang/go/wiki/CodeReviewComments#imports

Comment (p-v) [inline main.go:11]: Thanks. Added the fix.

**Comment by huytd** [comment]
I'm in freeze mode because it's holiday, so, please wait, I will be back and merge the code on Tuesday :trollface: thanks guys!

**Pull request merged**